src/lib/server/embeddingEndpoints/tei/embeddingEndpoints.ts

```diff
@@ -0,0 +1,65 @@
+import { z } from "zod";
+import type { EmbeddingEndpoint } from "../embeddingEndpoints";
+import { chunk } from "$lib/utils/chunk";
+
+export const embeddingEndpointTeiParametersSchema = z.object({
+	weight: z.number().int().positive().default(1),
```

Comment:
how is `weight` parameter used?

Reply:
It is used exactly like the endpoint of the llms, send requests to each endpoint in proportion to its weights